doc/HOWTO_forks.md

```diff
@@ -8,7 +8,14 @@ set up remote repositories on each side.
 
 _While in your fork's directory_
 
-`git remote add upstream git@github.com:tablexi/nucore-open.git` (You should only need to do this once.)
+You should only need to do these once:
+
+```
+git remote add upstream git@github.com:tablexi/nucore-open.git
+chmod +x ./bin/merge_describer
```

Comment:
I just switched to this branch and was able to run the command without needing to change the permissions.